app/javascript/src/NewService/components/FormElements/FormWrapper.jsx

```diff
@@ -29,7 +29,7 @@ const FormWrapper = (props: FormProps) => {
           type="submit"
           name="commit"
           value={submitText}
-          className="important-button create"/>
+          className="pf-c-button pf-m-primary create" />
```

Comment:
I think we should use the Button component from `@patternfly/react-core` here as well.